server/rpm/pbench-server.spec

```diff
@@ -0,0 +1,241 @@
+        crontab -u pbench -r
+    fi
+    rm -rf /%{installdir}
+fi
+
+%posttrans
+
+%files
+%defattr(644, pbench, pbench, 755)
+/%{installdir}/VERSION
+/%{installdir}/SEQNO
+/%{installdir}/SHA1
+/%{installdir}/html/static/VERSION
+/%{installdir}/html/static/package.json
+/%{installdir}/package.json
+
+/%{installdir}/lib/config/pbench-server-satellite.cfg.example
+/%{installdir}/lib/config/pbench-server.cfg.example
+/%{installdir}/lib/config/pbench-server-default.cfg
+
+/%{installdir}/lib/crontab
+/%{installdir}/lib/mappings
+/%{installdir}/lib/settings
+
+/%{installdir}/lib/python3.7/site-packages/pbench/common/configtools.py
+/%{installdir}/lib/python3.7/site-packages/pbench/__init__.py
+/%{installdir}/lib/python3.7/site-packages/pbench/server/indexer.py
+/%{installdir}/lib/python3.7/site-packages/pbench/server/report.py
+/%{installdir}/lib/python3.7/site-packages/pbench/server/s3backup/__init__.py
+
+/%{installdir}/bin/pbench-base.sh
+
+%defattr(755, pbench, pbench, 755)
+/%{installdir}/bin/pbench-config
+/%{installdir}/bin/pbench-server-activate-create-crontab
+/%{installdir}/bin/pbench-server-prep-shim-002
+/%{installdir}/bin/pbench-audit-server
+/%{installdir}/bin/pbench-backup-tarballs
+/%{installdir}/bin/pbench-verify-backup-tarballs
+/%{installdir}/bin/pbench-clean-up-dangling-results-links
+/%{installdir}/bin/pbench-copy-sosreports
+/%{installdir}/bin/pbench-index
+/%{installdir}/bin/pbench-reindex
+/%{installdir}/bin/pbench-unpack-tarballs
+/%{installdir}/bin/pbench-satellite-cleanup
+/%{installdir}/bin/pbench-satellite-state-change
+/%{installdir}/bin/pbench-remote-satellite-state-change
+/%{installdir}/bin/pbench-remote-sync-package-tarballs
+/%{installdir}/bin/pbench-dispatch
+/%{installdir}/bin/pbench-report-status
+/%{installdir}/bin/pbench-pp-status
+/%{installdir}/bin/pbench-sync-package-tarballs
+/%{installdir}/bin/pbench-sync-satellite
+/%{installdir}/bin/pbench-server-set-result-state
+/%{installdir}/bin/pbench-audit-server.sh
+/%{installdir}/bin/pbench-backup-tarballs.py
+/%{installdir}/bin/pbench-base.py
+/%{installdir}/bin/pbench-clean-up-dangling-results-links.sh
+/%{installdir}/bin/pbench-copy-sosreports.sh
+/%{installdir}/bin/pbench-dispatch.sh
+/%{installdir}/bin/pbench-index.py
+/%{installdir}/bin/pbench-reindex.py
+/%{installdir}/bin/pbench-report-status.py
+/%{installdir}/bin/pbench-satellite-cleanup.sh
+/%{installdir}/bin/pbench-satellite-state-change.py
+/%{installdir}/bin/pbench-server-prep-shim-002.py
+/%{installdir}/bin/pbench-sync-package-tarballs.sh
+/%{installdir}/bin/pbench-sync-satellite.sh
+/%{installdir}/bin/pbench-trampoline
+/%{installdir}/bin/pbench-unpack-tarballs.sh
+/%{installdir}/bin/pbench-verify-backup-tarballs.py
+/%{installdir}/bin/pbench-check-tb-age
+/%{installdir}/bin/pbench-check-tb-age.py
+/%{installdir}/bin/pbench-cull-unpacked-tarballs
+/%{installdir}/bin/pbench-cull-unpacked-tarballs.py
+
+/%{installdir}/lib/python3.7/site-packages/pbench/common/__init__.py
```

Comment:
Shouldn't this be `python3.6`?

Reply:
Ah, yes - it's 3.7 on my Fedora box, but it needs to be generalized to work everywhere.